nixos/modules/services/misc/gitlab.nix

```diff
@@ -14,6 +14,20 @@ let
                       else
                         pkgs.postgresql_13;
 
+  # gitlab 16.3.0 requires git 2.41.0, but nixos 23.05 is at 2.40.1
+  gitPackage = if
```

Comment:
why not use https://gitlab.com/gitlab-org/git/-/tags/v2.41.0.gl1?

Reply:
It doesn't compile